txpipe/twopoint.py

```diff
@@ -774,10 +909,65 @@ def plot_shear_density(self, s, sources, lenses):
                     fontsize=10, verticalalignment='top', bbox=props)
 
         plt.tight_layout()
-        plt.subplots_adjust(hspace=0,wspace=0)
+        plt.subplots_adjust(hspace=self.config['hspace'],wspace=self.config['wspace'])
```

Comment:
This also repeats a lot of the stuff above - possible to combine?

Reply:
@joezuntz do you mean in general combine all the plotting code or just this line?

Reply:
The general plotting code.  No need to go crazy if it's more awkward than I thought!

Reply:
Yeah I was thinking about that.. I agree it'd be much better, I'll give it a try